sale_rental/models/product.py

```diff
@@ -20,7 +20,7 @@ class ProductProduct(models.Model):
         string='Related Rental Services')
 
     @api.one
-    @api.constrains('rented_product_id', 'must_have_dates', 'type', 'uom_id')
+    @api.constrains('rented_product_id')
```

Comment:
Runbot is failing because of this:
![image](https://cloud.githubusercontent.com/assets/19620251/23303646/9739b0a6-fa95-11e6-9cd2-f76cf6985c2c.png)

This normally caused because the fields are readonly. It seems that this is not the case. I think it's better to discuss in the issue and close this then.